_config.yml

```diff
@@ -103,3 +103,9 @@ authors:
     email: ariel@ombulabs.com
     twitter: arieljuod
     github: arielj
+  fionadl:
+    display_name: "Fiona Lapham"
+    gravatar: 5d9d102c1890a27d8375933f20ebe647
```

Comment:
@FionaDL I think this is not the correct one. If you go to https://s.gravatar.com/avatar/5d9d102c1890a27d8375933f20ebe647 it doesn't show your photo

Reply:
Added a different one, let me know.